aspnetcore/includes/key-features.md

```diff
@@ -0,0 +1,53 @@
+Key features:
+
+:::moniker range=">= aspnetcore-6.0"
```

Comment:
Why do we need these version ranges if the text is the same?

Reply:
Im not at home right now to look; but IIRC, the 6.0 section has Minimal APIs, and the 3.0 to 6.0 bit doesn't.

We do need to version the whole bullet list to get good rendering. We can't version single bullets.

Reply:
Ah, your right, my visual diff failed to see the difference. Thanks!